R/extractVariableGroups.R

```diff
@@ -28,7 +35,8 @@ extractVariableGroups <- function(x) {
     matches <- grep(sep,x,fixed=TRUE, value = TRUE)
     if(length(matches)==0) break()
     ext <- ifelse(i>1,paste0(" ",i),"")
-    out <- c(out,tmp(matches,sep=sep,ext=ext))
+    if (keepOrigNames) ext<-""
```

Comment:
perhaps move that into the tmp function to have all modifications related to keepOrigNames in one place